test-one.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+. $NVM_DIR/nvm.sh
+
+version=$1
+nvm install $version
+node test.js
+node --es_staging test.js
+node --harmony test.js
+nvm deactivate
+nvm uninstall $version  # Don't fill the disk with node installs.
```

Comment:
This ended up still being required; the download cache is fine to be large, however, acutally storing the unpacked versions did hit a disk limit in my testing.